hack/generate/samples/internal/pkg/utils.go

```diff
@@ -34,19 +34,6 @@ func CheckError(msg string, err error) {
 	}
 }
 
-// CreateBundle runs all commands to create an operator bundle.
-func (ctx *SampleContext) CreateBundle() {
-	log.Infof("integrating project with OLM")
-	err := ctx.DisableManifestsInteractiveMode()
-	CheckError("disabling `generate kustomize manifests` interactive mode", err)
-
-	err = ctx.Make("bundle", "IMG="+ctx.ImageName)
-	CheckError("running make bundle", err)
-
-	err = ctx.StripBundleAnnotations()
-	CheckError("stripping bundle annotations", err)
-}
```

Comment:
note that we are moving it out to the `internal/testutils/olm.go` where it can be used by both.